scripts/metrics/junit_eval.py

```diff
@@ -111,8 +111,10 @@ def _main():
     taskname = metric_utils.get_taskname()
 
     # Get env variables
-    default_points = int(metric_utils.get_env_variable(
-        DEFAULT_POINTS_ENV_KEY, taskname, USAGE))
+    default_points = 1
```

Comment:
bin jetzt nicht so in Python drinne, 
aber in Java würde man ja sowas wie 

```
final int DEFAULT_POINTS=1;
...
int points; 
if metric_utils.has_env_variable(DEFAULT_POINTS_ENV_KEY, taskname):
        default_points = int(metric_utils.get_env_variable(
            DEFAULT_POINTS_ENV_KEY, taskname, USAGE))
else points=DEFAULT_POINTS;

```

machen. Also Konstante oben in der File definieren.

Reply:
Ist jetzt weiter oben im Skript als globale Variable definiert.